src/components/Tag.js

```diff
@@ -110,30 +120,41 @@ const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
               '?search[stringTraits][0][name]=' +
               title +
               '&search[stringTraits][0][values][0]=' +
-              text
+              value ?? text
           );
         }
       }
     );
-  }, [slug, text, title]);
+  }, [slug, text, value, title]);
+
+  const textWithUpdatedCase = keepTextLowerCase ? text : upperFirst(text);
```

Comment:
```suggestion
  const textWithUpdatedCase = lowercase ? text : upperFirst(text);
```
I would prefer this working

Reply:
👍